pkgs/development/interpreters/python/wrap.sh

```diff
@@ -75,7 +75,7 @@ wrapPythonProgramsIn() {
                                     --prefix PATH ':' "$program_PATH"
                                     )
 
-                    if [ -z "$permitUserSite" ]; then
+                    if [ -n "$permitUserSite" ]; then
                         wrap_args+=(--set PYTHONNOUSERSITE "true")
                     fi
```

Comment:
What is wrong here?

AIUI: `-z` returns true for non-zero length strings, so by default we enable PYTHON**NO**USERSITE.

Reply:
True. This is really confusing though.